test/verify/check-openshift

```diff
@@ -76,9 +76,12 @@ class TestOpenshift(MachineCase, OpenshiftCommonTests):
 
         wait_project(self.openshift, "marmalade")
 
+        self.openshift.execute("oc project default")
```

Comment:
And here it does it again anyway, so it shouldn't be done in the .setup script. What's the rationale for this? This wasn't necessary before.

Reply:
Added here due to testing. Can be removed if set in `openshift.setup`.
Anyway, the actual project is an assumption. Having it here prevents issues later and makes the flow more obvious.
Having it in `openshift.setup` or `kubevirt.setup` helps others when adding more stuff to the image.